Components/Queues/Drivers/ArrayQueueDriver.php

```diff
@@ -166,9 +171,26 @@ public function createQueueMessage()
     /**
      * Clean all the opened resources, must be called just before terminating the current request.
      */
-    public function doDestroy()
+    public function destroy()
     {
         // TODO: Implement doDestroy() method.
         // I have no time to do destroy the world.
     }
+
+    /**
+     * @return string
+     */
+    public function getFormat()
+    {
+        return $this->format;
+    }
+
+    /**
+     * @param string $format
+     * @return ArrayQueueDriver
```

Comment:
Thins function doesn't have any return, this doc shouldn't be needed

Reply:
FixEd.